### DISCUSSION TITLE
Docker webgui port in host mode

### DISCUSSION THREAD

**Discussion by lineumaciel** [openthread/openthread#8852] [Q&A] [answered]
Hi, when building a docker image what argument should be used to change the webgui port?

**Comment by jwhui** [comment, marked as answer]
See `OTBR_WEB_OPTS` in https://github.com/openthread/ot-br-posix/blob/394274e2afd9a4906c77d97a79790fc7a362b669/etc/docker/docker_entrypoint.sh#L100

**Reply by lineumaciel** [reply]
Thanks!
